__tests__/app.test.js

```diff
@@ -0,0 +1,18 @@
+const request = require('supertest')
+const app = require('../app')
+const data = require('../db/data/test-data/index')
+const seed = require('../db/seeds/seed')
+
+const categoriesData = require('../db/data/test-data/categories')
+
+beforeEach(() => {
+    return seed(data);
+})
+
```

Comment:
Please end the connection to your database once you're test has run

Reply:
will do